hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryEngineImpl.java

```diff
@@ -93,9 +93,14 @@ public Result execute(Query query, Target target) {
             case ALL_NODES:
                 return runOnAllPartitions(adjustedQuery);
```

Comment:
Seems `runOnAllPartitions` can also be removed and 
`runOnGivenPartitions(adjustedQuery, getAllPartitionIds(), TargetMode.ALL_NODES)` can be used instead?

Reply:
fixed